src/test/java/lesson6/task1/PrintTest.java

```diff
@@ -0,0 +1,21 @@
+package lesson6.task1;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PrintTest {
+
+    @Test
+    void shouldPrintFileContent() throws IOException {
+        //given
+        String expectedContent = "Course\nJava\nLesson 6\nFiles\nExceptions\nFiles\nEnd of file";
```

Comment:
If a run this test on my computer, test will fail, because, this file in somewhere outside project folder so you dont have it included in your repository. Am I right?